interfaces/builtin/polkit.go

```diff
@@ -172,16 +172,20 @@ func isPathMountedWritable(mntProfile *osutil.MountProfile, fsPath string) bool
 	return false
 }
 
+var (
+	// polkitProcSelfMounts is the path of /proc/self/mounts as used by polkit interface.
+	polkitProcSelfMounts = "/proc/self/mounts"
+	// polkitDaemonPath1 is the path of polkitd on core<24.
+	polkitDaemonPath1 = "/usr/libexec/polkitd"
```

Comment:
Nitpick, name could be more specific and include "core<24" in some form, same for other.

Reply:
I'd rather remove any reference to core as over time other distributions will look like one or the other mode and the specific binding to which distro it is does not really matter.